lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -1406,10 +1464,25 @@ def from_ansible_params(cls, ap, old_service, image_digest, can_update_networks)
             s.mounts = []
             for param_m in ap['mounts']:
                 service_m = {}
-                service_m['readonly'] = param_m['readonly']
+                service_m['read_only'] = param_m['readonly']
```

Comment:
I think this change should be documented. In case anyone uses the return value of the module, this might break their roles/playbooks.

Reply:
Do you think a change log entry will do or should we add it to the porting guide?

Reply:
One option is also to keep the returned key in the facts method as readonly.

Reply:
I think a change log entry will do. It's not a too important change IMO. If you want I can ask on #ansible-devel to be on the safe side.

Reply:
I agree that the change should be pretty low impact. I've added a changelog fragment.

Reply:
I looked through the PR again. Is this rename really necessary? So far, it looks like the values returned by the module (at least for the `mounts` option) are named exactly as the options passed *into* the module. This change will break this symmetry. About the loop with `mount_options` below (which I guess is one of the reasons why you changed it): you could change `mount_options` to a `dict`, mapping Ansible option name to docker-py's parameter name. (This approach is for example used in the `docker_container` module, where there are more discrepancies.)

Reply:
Yes you're right. Good idea with using a dict for mapping the args. I'll fix it.